css/sitesummary.css

```diff
@@ -11,3 +11,25 @@
 .gridfield-button-link {
     margin-bottom: 12px;
 }
+
+.cms table.ss-gridfield-table tr td.col-Summary {
+    padding: 0;
+}
```

Comment:
This will probably apply to any gridfield that has a Summary column. Are you able to use a parent selector that has "reports" in it somewhere? There's often the PHP class name somewhere in the top level of the DOM

Reply:
I figured this was page specific CSS so it didn't matter?

Reply:
If you view a report and then navigate to another area it may be loaded with AJAX and still have this in the browser

Reply:
I see. I've updated this. I ended up adding a class to the gridfield specifically. I couldn't seem to find something that was specific enough for just this report otherwise.

Reply:
Good move